pyproject.toml

```diff
@@ -13,6 +13,7 @@ boto3 = "^1.26.16"
 tqdm = "^4.64.1"
 aws-error-utils = "^2.7.0"
 pandas = "^1.5.3"
+datasets = "^2.10.1"
```

Comment:
this is an interesting decision point should we expand the data access lib to use features from more libraries – should we install the libraries specifically, or warn if the user hasn't got the library installed?

Reply:
hmm not sure, don't have a strong opinion.